lib/metadata/bucketfile/backend.js

```diff
@@ -8,7 +8,7 @@ import constants from '../../../constants';
 import config from '../../Config';
 
 const errors = arsenal.errors;
-const MetadataClient = arsenal.storage.metadata.client;
+const MetadataClient = arsenal.storage.metadata.MetadataFileClient;
```

Comment:
Or more exactly, it depends on https://github.com/scality/Arsenal/pull/235 which itself has a two-way dependency with #686.